### ISSUE TITLE
Parser fails with "except" (BigQuery)

### ISSUE THREAD

**Issue by jml** [alanmcruickshank/sqlfluff#329]
```console
➜ sqlfluff version
0.3.4
```

Config:

```ini
[sqlfluff]
dialect = bigquery
templater = jinja

[sqlfluff:rules]
tab_space_size = 2
```

The following SQL fails lint checks with a parser error:

```sql
select
  * except (foo)
from some_table

  union all

select
  *
from another_table
```

Here's the output:

```console
➜ sqlfluff lint test.sql
== [test.sql] FAIL
L:   5 | P:   3 | L011 | Implicit aliasing of table not allowed. Use explicit
                       | `AS` clause.
L:   5 | P:   3 | L025 | Alias 'union' is never used in SELECT statement.
L:   5 | P:   3 | L029 | Keywords should not be used as identifiers.
L:   5 | P:   8 |  PRS | Found unparsable section: ' all\n\nselect\n *\nfrom
                       | another_table'
```

Whereas this SQL passes all checks:

```sql
select
  *
from some_table

union all

select
  *
from another_table
```

**Comment by alanmcruickshank** [comment]
thanks for an excellent issue report! I'm tempted to make this into a template.

Agreed. This hasn't been implemented but can be easily. Watch this space.

**Comment by alanmcruickshank** [comment]
Ah - this one is harder than expected. Problem is that `EXCEPT` is both a set operator and a modifier to select....
